YCloudYUSA/y_lb
# DS-1582 fix: Set color for entity browser modal close button

https://yusa.atlassian.net/browse/DS-1582

To test:
- Add LB block with an image
- Add an image.
- Observe modal close block is visible.

<img width="211" alt="SCR-20240722-oekj" src="https://github.com/user-attachments/assets/c427b26a-ff49-4388-9857-8044b7239832">
